shared/js/background/abp-lists.es6.js

```diff
@@ -12,6 +12,7 @@ const surrogates = require('./surrogates.es6')
 const settings = require('./settings.es6')
 const ATB = require('./atb.es6')
 const load = require('./load.es6')
+const polyfill = require('./safari-polyfill.es6')
```

Comment:
I've been thinking about the "polyfill" name. I'm not sure "polyfill" is quite there. You could say the Safari is kind of a polyfill for features that the webextensions API normally supports, but in that case the name doesn't work for Chrome and FF.

How about something like "[chrome/safari]-wrapper"? "[chrome/safari]-extension-utils"? Or just "[chrome/safari]-utils"?